docs/tasks/uninstall/kubernetes.rst

```diff
@@ -0,0 +1,85 @@
+==========================
+Uninstalling on Kubernetes
+==========================
+
+Below is the processes for uninstalling cert-manager on Kubernetes. There are
+two processes to chose depending on which method you used to install
+cert-manager - static manifests or ``helm``.
+
+.. warning::
+
+   To uninstall cert-manger you should always use the same process for installing
+   but in reverse. Deviating from the following process whether cert-manager has
+   been installed from static manifests or helm can cause issues and
+   potentially broken states. Please ensure you follow the below steps when
+   uninstalling to prevent this happening.
+
+Before continuing, ensure that all cert-manager resources that have been created
+by users have been deleted. You can check for any existing resources with the
+following command:
+
+.. code-block:: shell
+
+   kubectl get Issuers,ClusterIssuers,Certificates,CertificateRequests --all-namespaces
```

Comment:
`Orders` and `Challenges` too?

Reply:
Hm, surely they will be garbage collected though?
They are not something a user creates too

Reply:
A user *could* create them though - I feel like we're safest to copy everything we create 😄

Reply:
but yeah, most of them *will* be deleted as they'll be owned by a resource that no longer exists